source/docs/essentials/extend-the-graphql-schema.md

```diff
@@ -1,23 +1,14 @@
 ---
 id: extend-the-graphql-schema
 title: Extend the GraphQL schema
+description:
+  When developing an e-commerce store, you might at some point need to expose new data in your unified GraphQL schema to support new features and allow frontend developers to use them.
+  Front-Commerce’s GraphQL modules  is the mechanism allowing to extend and override any part of the schema defined by other modules.
```

Comment:
Same comment for multiline descriptions 🔁

Reply:
Done. Thanks.